### ISSUE TITLE
Upgrade from 2.10.10 to Latest is Failing

### ISSUE THREAD

**Issue by Wasca** [netbox-community/netbox-docker#529]
## Current Behavior

Followed the Update instructions on this page - https://github.com/netbox-community/netbox-docker/wiki/Updating

After bringing up the stack, the netboxdocker_netbox_1 container is showing an error while trying to apply the migrations.

If I visit the URL where my netbox install is location I get an "Unable to connect" Error in the browser

## Expected Behavior

I expected that my netbox docker stack would process the upgrade and be running as per normal.

## Debug Information

The output of `docker-compose version`: `1.28.6`
The output of `docker version`: `20.10.6`
The output of `git rev-parse HEAD`: `676ca798cd684ce30fca2692180e680d2becf76e`
The command you used to start the project: `I use portainer to start the stack`

The output of `docker inspect netboxcommunity/netbox:latest --format "{{json .Config.Labels}}"`:

```json
{
   "BUILD_REASON":"alpine netbox netbox-docker",
   "NETBOX_GIT_BRANCH":"HEAD",
   "NETBOX_GIT_REF":"6c1f9dba52c2c0e9a19a53436c2394c1808ccd3a",
   "NETBOX_GIT_URL":"https://github.com/netbox-community/netbox.git",
   "ORIGINAL_TAG":"docker.io/netboxcommunity/netbox:v2.11.4",
   "org.label-schema.build-date":"2021-05-26T06:20+00:00",
   "org.label-schema.description":"A container based distribution of NetBox, the free and open IPAM and DCIM solution.",
   "org.label-schema.name":"NetBox Docker",
   "org.label-schema.schema-version":"1.0",
   "org.label-schema.url":"https://github.com/netbox-community/netbox-docker",
   "org.label-schema.usage":"https://github.com/netbox-community/netbox-docker/wiki",
   "org.label-schema.vcs-ref":"676ca798cd684ce30fca2692180e680d2becf76e",
   "org.label-schema.vcs-url":"https://github.com/netbox-community/netbox-docker.git",
   "org.label-schema.vendor":"The netbox-docker contributors.",
   "org.label-schema.version":"1.2.0",
   "org.opencontainers.image.authors":"The netbox-docker contributors.",
   "org.opencontainers.image.created":"2021-05-26T06:20+00:00",
   "org.opencontainers.image.description":"A container based distribution of NetBox, the free and open IPAM and DCIM solution.",
   "org.opencontainers.image.documentation":"https://github.com/netbox-community/netbox-docker/wiki",
   "org.opencontainers.image.licenses":"Apache-2.0",
   "org.opencontainers.image.revision":"676ca798cd684ce30fca2692180e680d2becf76e",
   "org.opencontainers.image.source":"https://github.com/netbox-community/netbox-docker.git",
   "org.opencontainers.image.title":"NetBox Docker",
   "org.opencontainers.image.url":"https://github.com/netbox-community/netbox-docker",
   "org.opencontainers.image.vendor":"The netbox-docker contributors.",
   "org.opencontainers.image.version":"1.2.0"
}
```

The output of `docker-compose logs netbox`:

Log Files here - https://gist.github.com/Wasca/4e7456d70250fd087b5d68ed8f780095

The output of `cat docker-compose.override.yml`:

```text
version: '3.4'
services:
  netbox:
    ports:
      - 8001:8080
```

**Comment by cimnine** [comment]
I don't see that this error has something to do with our Dockerized version of NetBox.

One thing you could try is to shorten the update step, i.e. update from v2.10.10 -> v2.11.0 first, then `v2.11.0` -> `v2.11.1`, etc., and see how this works out.

Also I suggest to use specific versions rather than `latest`. Because of the way the Docker cache works you need to ensure manually that `latest` is the actual latest version of the image, and not some `latest` that was previously the latest version, but which has now been superseeded.

**Comment by cimnine** [comment]
I will close this issue. If you believe that this problem has something to do with how we package NetBox in a Docker image, then please feel free to open the issue again.

**Comment by Wasca** [comment]
@cimnine I have removed all the old images and updated my docker-compose.yml file to point to `image: netboxcommunity/netbox:${VERSION-v2.11.0}` I then start the stack and I'm getting the same errors as I have already outlined in my log files above. 

It seems that what ever process you're using to upgrade an existing v2.10.10 netbox-docker install is failing.

This error is what is killing the upgrade process. Refer to this log file [netboxdocker_netbox_1-log](https://gist.github.com/Wasca/4e7456d70250fd087b5d68ed8f780095#file-netboxdocker_netbox_1-log) 

I'm not sure sure what else I can do.
